### ISSUE TITLE
views::ints(lower) is deprecated but not documented thus / views::iota(lower) is not

### ISSUE THREAD

**Issue by dtkb** [ericniebler/range-v3#1534]
![image](https://user-images.githubusercontent.com/63455896/88174069-8671a700-cc1b-11ea-8ba9-45f5ab647a45.png)

The documentation indicates that not supplying an upper bound is fine. But the code deprecates this for `ints`:

![image](https://user-images.githubusercontent.com/63455896/88174119-a1dcb200-cc1b-11ea-90ca-0ee32deb4555.png)

No such deprecation is made for `iota`:

![image](https://user-images.githubusercontent.com/63455896/88174168-bae56300-cc1b-11ea-8cd7-784e18dffbc6.png)

Should it be?

In any case, the documentation presumably shouldn't so enthusiastically indicate that the deprecated overload of `ints` is OK.

**Comment by johelegp** [comment]
No, because it's part of the standard. `std::views::iota(n)` [is OK](https://timsong-cpp.github.io/cppwp/n4861/range.iota#view-itemdecl:3).

**Comment by ericniebler** [comment]
For `iota`, which works with any `incrementable` type including iterators, there is only one possible interpretation for `iota(i)` ... an infinite range starting at `i`. For `ints`, the story is different. There is ambiguity about meaning of `ints(N)`. It could be an infinite range starting at `N`, or a half-open range starting at `0` and ending at `N-1`. Since it's ambiguous and error-prone, it is deprecated.

The docs can use some updating.
